src/main.ts

```diff
@@ -39,10 +39,76 @@ async function main() {
 
 	logger.info(`Version: ${packageJSON.version}`);
 
+	// curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "state_traceBlock", "params": ["0xb246acf1adea1f801ce15c77a5fa7d8f2eb8fed466978bcee172cc02cf64e264"]}' http://localhost:9933/
+	const rpc = {
+		state: {
+			traceBlock: {
+				description: 'Blcok state trace prototype',
+				params: [
+					{
+						name: 'at',
+						type: 'BlockHash',
+						isOptional: true,
+						isHistoric: true,
+					},
+					{
+						name: 'targets',
+						type: 'Text',
+						isOptional: true,
+						isHistoric: true,
+					},
+					{
+						name: 'storage_keys',
+						type: 'Text',
+						isOptional: true,
+						isHistoric: true,
+					},
+				],
+				type: 'Json',
+			},
+		},
+	};
+
+	// Note: These are not working - currently just using `Json` type
+	// const _types = {
```

Comment:
Remove these commented out types

Reply:
Friendly reminder that these have to get removed.